airbyte-integrations/connectors/source-pipedrive/pyproject.toml

```diff
@@ -16,8 +16,8 @@ repository = "https://github.com/airbytehq/airbyte"
 include = "source_pipedrive"
 
 [tool.poetry.dependencies]
-python = "^3.9,<3.12"
-airbyte-cdk = "^0"
+python = "^3.10,<3.12"
+airbyte-cdk = "^6"
```

Comment:
small reminder to re-run `poetry lock` since we now have version `6.9.0` which included some CDK fixes to concurrent that I added

Reply:
updated poetry.lock file